JS_HW_1/js_hw_1_task_3.js

```diff
@@ -0,0 +1,14 @@
+
+//  Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную n + nn + nnn, где n не перемножаются, а конкатенируются
+
+function randomN(min, max) {
+    let n = min + Math.random() * (max + 1 - min);
+    return Math.floor(n);
+  };
+  console.log(
+    randomN(1, 9)  
```

Comment:
As per requirements we must use only one number, but each call of randomN function returns new value. Need to store created one in a variable and use.

Reply:
@anatoly-karpovich Thank you for your comments! I've updated the documents and added them to the branch.